src/Markdown.MAML/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="YamlDotNet" version="4.2.1" targetFramework="net45" />
+</packages>
```

Comment:
Each project needs it's own packages.config for the things it depends on (just like the test project has the xunit libraries). You can get rid of the .nuget/* files if your project is running on .Net Core in VS 2017 since NuGet is built in to the new tool chain but this is still the proper way to do things for .Net Framework projects